MAGECLOUD-4099: Check how Mutagen (https://mutagen.io) works with Windows (#5860)
magento/devdocs: guides/v2.2/cloud/docker/docker-config.md

    ```bash
    bash ./mutagen.sh
    ```
    
    {: .bs-callout-info}

>     If you running docker on Windows and session start was failed, change `--symlink-mode` to `portable` in the `mutagen.sh` file.

Reviewer: ```suggestion
   If you use `docker-sync` for file synchronization, skip this step.  If you host your Docker environment on Windows and the session start fails, update the `mutagen.sh` file to change the  `--symlink-mode` to `portable`.
```

Author: They are described in the mutagen documentation https://mutagen.io/documentation/synchronization/symbolic-links/
Maybe we can put a link into our devdocs.

Reviewer 2: I see! Okay, don't worry about that here, then. I will consider it in the changes I am making for later. Thanks!

>     {: .bs-callout-info}

Reviewer: ```suggestion
```

Author: I accepted another suggestion and can't accept this as the file was changed.
Removed `{: .bs-callout-info}` manually.

>     If you use `docker-sync` for file synchronization, skip this step.

Reviewer: ```suggestion
```

Reviewer: Recommend combining the two notes since they are in the same step. I know that @hguthrie is reworking the Docker content, so she might have some additional feedback on this PR.

Reviewer: @oshmyheliuk  If you can make the following updates, then we can merge this PR.  Heather can incorporate additional updates in the PR that she's creating to improve  the Docker content.

Accept the suggestions to update the wording on the note you added and remove the second note.  Then, change line 221 to: 

```
1. Start the file synchronization with mutagen.io. If you use docker-sync for file synchronization, skip this step.
```
